apps/autorelayer-interop/src/relayer.ts

```diff
@@ -59,7 +59,6 @@ export class Relayer {
         destination: message.destination,
         messageHash: message.messageHash,
         txHash: message.transactionHash,
-        txOrigin: message.txOrigin,
```

Comment:
removed this only bc it can be inspected by looking up the txHash on an explorer. the context is just a minimal set of info